torch/_inductor/codegen/cpp_prefix.h

```diff
@@ -58,41 +58,56 @@ struct IsVecType<at::vec::Vectorized<T>>: std::true_type {};
 #endif
 
 template <typename T>
-Welford<T> welford_combine(const Welford<T> &a, const Welford<T> &b) {
-  if constexpr (!IsVecType<T>::value) {
-    if (a.weight == 0) {
-      return b;
-    }
-    if (b.weight == 0) {
-      return a;
+struct WeightRecp {
+  int64_t N;
+  std::vector<T> weight_recps;
```

Comment:
Is this array of `Vectorized<float>` necessary for performance? I would think the scalar `float` would be good enough and require less memory overhead.

Reply:
My gut feeling is that scalar float should be good enough.

Reply:
Used scalar instead.